src/test/java/org/dinosaur/foodbowl/file/FileTestUtils.java

```diff
@@ -0,0 +1,59 @@
+package org.dinosaur.foodbowl.file;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+public class FileTestUtils {
+
+    private static final String TEST_FILE_UPLOAD_PATH = "src/test/resources/bucket";
+
+    public static List<MultipartFile> generateMultipartFiles(int size) {
+        List<MultipartFile> multipartFiles = new ArrayList<>();
+
+        for (int i = 0; i < size; i++) {
+            multipartFiles.add(new MockMultipartFile(
+                    "images",
+                    "foodBowl.jpg",
+                    MediaType.TEXT_PLAIN_VALUE,
+                    "Hello Images".getBytes()));
+        }
```

Comment:
```suggestion
        for (int i = 0; i < size; i++) {
            MockMultipartFile file = new MockMultipartFile(
                    "images",
                    "foodBowl.jpg",
                    MediaType.TEXT_PLAIN_VALUE,
                    "Hello Images".getBytes()
            );
            multipartFiles.add(file);
        }
```
사소하지만 요런 코드 컨벤션은 어떤가요? 😙

Reply:
```suggestion
        for (int i = 0; i < size; i++) {
            MockMultipartFile file = generateMockMultiPartFile();
            multipartFiles.add(file);
        }
```
아래의 메서드도 재활용한다면 이렇게도 할 수 있을 것 같아요 :)